diff -urNp -x '*.orig' bftpd.org/Makefile.in bftpd/Makefile.in
--- bftpd.org/Makefile.in	2009-10-06 23:27:39.000000000 +0200
+++ bftpd/Makefile.in	2023-02-02 09:01:57.425707183 +0100
@@ -26,14 +26,10 @@ bftpd: $(OBJS)
 $(OBJS): $(HEADERS) Makefile
 
 install: all
-	$(INSTALL) -g 0 -m 700 -o 0 bftpd $(DESTDIR)/$(prefix)/sbin
-	$(INSTALL) -g 0 -m 644 -o 0 bftpd.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL) bftpd $(DESTDIR)/$(prefix)/sbin
+	$(INSTALL) bftpd.8 $(DESTDIR)/$(mandir)/man8
 	[ -f $(DESTDIR)/etc/bftpd.conf ] || \
-		$(INSTALL) -g 0 -m 600 -o 0 bftpd.conf $(DESTDIR)/etc
-	touch /var/log/bftpd.log
-	chmod 644 /var/log/bftpd.log
-	mkdir -p /var/run/bftpd
-	chmod 755 /var/run/bftpd
+		$(INSTALL) bftpd.conf $(DESTDIR)/etc
 
 clean distclean:
 	rm -f *~ $(OBJS) bftpd mksources.finished config.cache
