--- bftpd-1.0.24/Makefile.in.orig	Mon May 26 16:00:16 2003
+++ bftpd-1.0.24/Makefile.in	Mon May 26 16:02:53 2003
@@ -26,14 +26,10 @@
 $(OBJS): $(HEADERS) Makefile
 
 install: all
-	$(INSTALL) -g 0 -m 700 -o 0 bftpd $(DESTDIR)/$(prefix)/sbin
-	$(INSTALL) -g 0 -m 644 -o 0 bftpd.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL) bftpd $(DESTDIR)/$(prefix)/sbin
+	$(INSTALL) bftpd.8 $(DESTDIR)/$(mandir)/man8
 	[ -f $(DESTDIR)/etc/bftpd.conf ] || \
-		$(INSTALL) -g 0 -m 600 -o 0 bftpd.conf $(DESTDIR)/etc
-	touch /var/log/bftpd.log
-	chmod 644 /var/log/bftpd.log
-	mkdir -p /var/run/bftpd
-	chmod 755 /var/run/bftpd
+		$(INSTALL) bftpd.conf $(DESTDIR)/etc
 
 clean distclean:
 	rm -f *~ $(OBJS) bftpd mksources.finished
