diff -ur bftpd-1.0.15.orig/Makefile.in bftpd-1.0.15/Makefile.in
--- bftpd-1.0.15.orig/Makefile.in	Tue Jan  2 18:46:58 2001
+++ bftpd-1.0.15/Makefile.in	Sat Jan 13 20:44:49 2001
@@ -19,11 +19,9 @@
 $(OBJS): $(HEADERS) Makefile
 
 install: all
-	$(INSTALL) -g 0 -m 700 -o 0 bftpd $(DESTDIR)/$(prefix)/sbin
-	$(INSTALL) -g 0 -m 644 -o 0 bftpd.8 $(DESTDIR)/$(mandir)/man8
-	$(INSTALL) -g 0 -m 600 -o 0 bftpd.conf $(DESTDIR)/etc
-	touch /var/log/bftpd.log
-	chmod 644 /var/log/bftpd.log
+	$(INSTALL) -m 700 bftpd $(DESTDIR)/$(prefix)/sbin
+	$(INSTALL) -m 644 bftpd.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL) -m 600 bftpd.conf $(DESTDIR)/etc
 
 clean distclean:
 	rm -f *~ $(OBJS) bftpd
